Plugins/Onnx/include/Acts/Plugins/Onnx/MLTrackClassifier.hpp

```diff
@@ -0,0 +1,49 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include <vector>
+
+#include "Acts/Plugins/Onnx/OnnxRuntimeBase.hpp"
+
+namespace Acts {
+
+// Specialized class that labels tracks as good/duplicate/fake using a
+// deep neural network.
+class MLTrackClassifier : public OnnxRuntimeBase {
+  using OnnxRuntimeBase::OnnxRuntimeBase;
+
+ public:
+  /// @brief The labels for track quality
+  enum TrackLabels { good, duplicate, fake };
+
+  /// @brief Predict the track label
+  ///
+  /// @param inputFeatures The vector of input features for the trajectory to be
+  /// classified
+  /// @param decisionThreshProb The probability threshold used to predict the
+  /// track label
+  ///
+  /// @return The predicted track label of the trajectory
+  TrackLabels predictTrackLabel(std::vector<float>& inputFeatures,
+                                const double& decisionThreshProb) const;
```

Comment:
```suggestion
                                double decisionThreshProb) const;
```
Nitpicking: scalar values should be handed over by-value.